test/fixtures/basic_answers.yaml

```diff
@@ -1 +1,10 @@
-puppet: true
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# See params.pp in each class for what options are available
+
+--- 
+  puppet: true
```

Comment:
This file is unfortunately updated during the tests. This is how config.store(@data) wrote it. I'm thinking of moving answer file to temp file so the fixture is not changed.